Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -19,6 +19,10 @@ class LoginsHelper: BrowserHelper {
     private let profile: Profile
     private var snackBar: SnackBar?
 
+    var logins: BrowserLogins {
+        return profile.logins
```

Comment:
Yes and no. The only reason I exposed this is to make it easier to provide a mock for the AuthenticatorTests. The alternative is to mock the entire LoginsHelper and use that but that relies on a Browser instance, etc.